modules/test/playwright/tests/portal-search-web/config.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {devices} from '@playwright/test';
+
+export const config = {
+	name: 'portal-search-web',
+	testDir: 'tests/portal-search-web',
+	timeout: 60 * 1000,
```

Comment:
Is this timeout needed? It looks like the default is already 90 seconds https://github.com/liferay-search/liferay-portal/blob/98dfcf7623cc9d0de9fa9f54a06e648341458009/modules/test/playwright/playwright.config.ts#L102

Reply:
Oops, I must have copied this from somewhere. Taking it out 👍